mp2-v1/src/final_extraction/base_circuit.rs

```diff
@@ -113,6 +113,8 @@ pub(crate) struct BaseCircuitProofWires {
 
 pub(crate) const CONTRACT_SET_NUM_IO: usize = contract_extraction::PublicInputs::<F>::TOTAL_LEN;
 pub(crate) const VALUE_SET_NUM_IO: usize = values_extraction::PublicInputs::<F>::TOTAL_LEN;
+// CHORE: Remove this when relevant PR is merged
+#[allow(dead_code)]
```

Comment:
This constant could be erased if unused I think

Reply:
Okay cool, just didn't want to get rid of it if it was about to be used in a PR thats in Review.